### PR TITLE
Adding tko-saas-deploy-guide 2.0

### PR DESCRIPTION
**Pull request by manishjha86**
Signed-off-by: Manish Jha <er.manish.jha@gmail.com>

### CODE REVIEW

File: wordlist.txt [hunk 1]
```diff
@@ -60,3 +60,4 @@ VNET
 TKG
 NSXT
 autoscaler
+configproxy
```
Comment (craigtracey) [inline wordlist.txt:63]: Remove this please.

File: src/deployment-guides/tko-saas-services.md [hunk 2]
```diff
@@ -35,100 +35,158 @@ Before managing Tanzu Kubernetes Grid clusters lifecycle from Tanzu Mission Cont
...
-13.  Click **View your Cluster** to see the cluster details.  
-    ![](./img/tko-saas-services/image83.png)
+2.  Click **View your Cluster** to see the cluster details.  
+    ![View Cluster Health](img/tko-saas-services/attach-workload-cluster06.png)
 
-### <a id=appendix-a> </a>Appendix (A) -  Create a Proxy Configuration Object in Tanzu Mission Control
+___
 
-Do the following steps to create a proxy configuration object in Tanzu Mission Control.
+  ### <a id=configproxy> </a>Supplemental Information: Create a Proxy Configuration Object in Tanzu Mission Control
```
Comment (craigtracey) [inline src/deployment-guides/tko-saas-services.md:156]: Please make this `<a id="configproxy">`

File: src/deployment-guides/tko-saas-services.md [hunk 3]
```diff
@@ -35,100 +35,158 @@ Before managing Tanzu Kubernetes Grid clusters lifecycle from Tanzu Mission Cont
...
+
+6.  Copy the provided command, switch to the bootstrap machine or any other machine which has kubectl/tmc installed and has connectivity to the management cluster, and then run the command.
+
+	- To obtain admin `kubeconfig` of a cluster, execute the following command on the bootstrap machine:
+
+		`tanzu cluster kubeconfig get <cluster-name> -n <namespace> --admin`
+
+	     Example:
+
+	     <!-- /* cSpell:disable */ -->
```
Comment (craigtracey) [inline src/deployment-guides/tko-saas-services.md:67]: Please remove this cSpell annotation

File: src/deployment-guides/tko-saas-services.md [hunk 4]
```diff
@@ -35,100 +35,158 @@ Before managing Tanzu Kubernetes Grid clusters lifecycle from Tanzu Mission Cont
...
+         ```
+         tanzu cluster kubeconfig get tkg-mgmt -n tkg-system --admin
+
+        Credentials of cluster 'tkg-mgmt' have been saved
+
+        You can now access the cluster by running 'kubectl config use-context tkg-mgmt-admin@tkg-mgmt'
+	    ```
+	    <!-- /* cSpell:enable */ -->
```
Comment (craigtracey) [inline src/deployment-guides/tko-saas-services.md:75]: Please remove this cSpell annotation

**Review by craigtracey — CHANGES_REQUESTED** [review]

**Review by srinivasme21 — APPROVED** [review]
IX review provided and merged.

**Review by craigtracey — APPROVED** [review]

**Pull request merged**